src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ClassUseWriter.java

```diff
@@ -144,6 +144,7 @@ public ClassUseWriter(HtmlConfiguration configuration,
 
         methodSubWriter = new MethodWriterImpl(this);
         constrSubWriter = new ConstructorWriterImpl(this);
+        constrSubWriter.setFoundNonPubConstructor(true);
```

Comment:
Yes, it is related. Member tables in the class-use page generally have three columns, while the constructor tables used to have only two (no modifier column). This is to make sure the tables for constructors also consistently have three columns. 

I have to admit I took the short path here by changing the number of columns displayed for constructor class-use tables, because keeping it at two columns would have required to override all or parts of the AbstractMemberWriter#addUseInfo method in ConstructorWriterImpl. I could still do that, but I didn't think it was worth the effort. 

Would it be ok if I added a comment to explain the reason for the line added above?